app/experimenter/experiments/tests/api/v5/test_serializers.py

```diff
@@ -898,46 +898,6 @@ def test_population_percent_bounds_check(self, expected_valid, population_percen
         else:
             self.assertNotIn("population_percent", serializer.errors)
 
-    @parameterized.expand(
-        [
-            [NimbusExperiment.Status.DRAFT, NimbusExperiment.Status.PREVIEW],
-            [NimbusExperiment.Status.DRAFT, NimbusExperiment.Status.REVIEW],
-            [NimbusExperiment.Status.PREVIEW, NimbusExperiment.Status.DRAFT],
-            [NimbusExperiment.Status.PREVIEW, NimbusExperiment.Status.REVIEW],
-        ]
-    )
-    def test_valid_status_update(self, from_status, to_status):
-        experiment = NimbusExperimentFactory(status=from_status)
-        serializer = NimbusExperimentSerializer(
-            experiment,
-            data={"status": to_status},
-            context={"user": self.user},
-        )
-        self.assertEqual(experiment.changes.count(), 0)
-        self.assertTrue(serializer.is_valid())
-        experiment = serializer.save()
-        self.assertEqual(experiment.changes.count(), 1)
-        self.assertEqual(experiment.status, to_status)
-
-    def test_status_with_invalid_target_status(self):
-        experiment = NimbusExperimentFactory(status=NimbusExperiment.Status.DRAFT)
-        serializer = NimbusExperimentSerializer(
-            experiment,
-            data={"status": NimbusExperiment.Status.ACCEPTED},
-            context={"user": self.user},
-        )
-        self.assertEqual(experiment.changes.count(), 0)
-        self.assertFalse(serializer.is_valid())
-        self.assertEqual(
-            serializer.errors,
-            {
-                "status": [
-                    "Nimbus Experiment status cannot transition from Draft to Accepted."
-                ]
-            },
-            serializer.errors,
-        )
-
```

Comment:
Deleted these because it's actually pretty well tested in TestNimbusStatusTransitionValidator

Reply:
Yaaaaa we definitely don't need to test it twice, but in cases like this where we have a mixin, we test it atomically, and then mix it into a consuming class, it's usually a good idea to have just a single integration test that will fail if you took the mixin out of the inheritance list.  If the mixin tests are testing directly against the consuming class then that's fine.